doc/ledger3.texi

```diff
@@ -5936,7 +5936,7 @@ Ledger does not expand any aliases if this option is specified.
 Accounts, tags or commodities not previously declared will cause errors.
 
 @item --permissive
-FIX THIS ENTRY @c FIXME thdox
+Quiet balance assertions.
```

Comment:
Please add a `[p][x]ref` to the _Balance assertions_ node it would be quite helpful for someone not knowing what balance assertions in Ledger are.